modules/seqexec/server/src/main/scala/seqexec/server/package.scala

```diff
@@ -85,6 +88,7 @@ package object server {
     Eq[Int].contramap(_.ordinal())
 
   val CalibrationQueueName: String = "Calibration Queue"
+  val CalibrationQueueId: UUID = UUID.fromString("7156fa7e-48a6-49d1-a267-dbf3bbaa7577")
```

Comment:
Why the specific UUID?

Reply:
Because it is a predefined queue. I thought it would be easier for the client implementation if the id of the calibration queue was known.